### PR TITLE
Add shipping dashboard endpoints

### PR DESCRIPTION
**Pull request by JoeArmani**
(1) Add biospecimen shipping dashboard endpoints for optimized box and specimen loading.
(2) Increase Box capacity to 40 bags.
(3) Update tube types for new streck tube implementation.
(4) Update submitParticipantData.json and updateParticipantData.json for new & expanded API-access.

Related issues:
Shipping Dashboard slow loading: https://github.com/episphere/connect/issues/619
Make specimen search faster: https://github.com/episphere/connect/issues/650 &
https://github.com/episphere/connect/issues/746
Increase max bag capacity: https://github.com/episphere/connect/issues/746
Add streck tubes support in ConnectFaas: https://github.com/episphere/connect/issues/620
Add new variable to rules JSON: https://github.com/episphere/connect/issues/739
Age expansion updated CIDs https://github.com/episphere/connect/issues/761

### CODE REVIEW

File: utils/biospecimen.js [hunk 1]
```diff
@@ -335,22 +315,34 @@ const biospecimenAPIs = async (req, res) => {
...
+            
+            if (requestData[fieldMapping.shippingBoxId]){
+                const boxId = requestData[fieldMapping.shippingBoxId];
+                const loginSite = requestData[fieldMapping.loginSite];
+                const addedTubes = requestData['addedTubes'];
+        
+                if (!addedTubes || addedTubes.length === 0) return res.status(400).json(getResponseJSON('Missing added tubes', 400));
+
+                const exists = await boxExists(boxId, loginSite, requestData);
+                if (exists !== true) return res.status(400).json(getResponseJSON('Box does not exist!', 400));
```
Comment (bransteitterbr) [inline utils/biospecimen.js:333]: I am not sure if the front end cares or needs to differentiate easily between the invalid request above and the not found here but normally I would expect this to be a 404 error

Comment (JoeArmani) [reply utils/biospecimen.js]: Good call. The front end doesn't care, but this is accurate. Updated. Thanks.

**Review by bransteitterbr — APPROVED** [review]

**Comment by bransteitterbr** [comment]
Looks good

**Review by anthonypetersen — APPROVED** [review]
very nice - looks good

**Pull request merged**